config/webpack.config.prod.js

```diff
@@ -398,7 +398,7 @@ module.exports = {
         "'self'",
         "https://*.comicrelief.com",
         "https://*.sportrelief.com",
-        "https://sentry.io",
+        // "https://sentry.io",
```

Comment:
I feel like we can leave these in place here; I don't think it will cause an issue if we're not even using Sentry?

Reply:
Yeah true